pkg/apis/apiserver/organization_REST.go

```diff
@@ -106,19 +123,221 @@ func (o *OrganizationREST) NewList() runtime.Object {
 }
 
 func (o *OrganizationREST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
-	return fakeOrg, nil
+	visible, err := o.isVisible(ctx, name, "get")
+	if err != nil {
+		return nil, err
+	}
+	if !visible {
+		return nil, apierrors.NewNotFound(qualifiedResource, name)
+	}
+
+	org, err := o.dynamicRI.Get(ctx, name, *options)
+	if err != nil {
+		return nil, err
+	}
+	return ConvertFromV1Alpha1Unstructured(org, o.scheme)
 }
 
 func (o *OrganizationREST) List(ctx context.Context, options *internalversion.ListOptions) (runtime.Object, error) {
-	// TODO: Replace with a read implementation
-	// small test whether the client is properly injected
-	s := &v1.NamespaceList{}
-	if err := o.client.List(ctx, s); err != nil {
+	opts := &metav1.ListOptions{}
+	if err := o.scheme.Convert(options, opts, nil); err != nil {
 		return nil, err
 	}
-	return &OrganizationList{Items: []Organization{*fakeOrg}}, nil
+	orgs, err := o.dynamicRI.List(ctx, *opts)
+	if err != nil {
+		return nil, err
+	}
+	sol, err := ConvertFromV1Alpha1UnstructuredList(orgs, o.scheme)
+	if err != nil {
+		return nil, err
+	}
+
+	lst := sol.Items
+	sol.Items = nil
+	for _, it := range lst {
+		visible, err := o.isVisible(ctx, it.Name, "get")
+		if err != nil {
+			return nil, err
+		}
+		if visible {
+			sol.Items = append(sol.Items, it)
+		}
+	}
+	return sol, nil
 }
 
 func (o *OrganizationREST) ConvertToTable(ctx context.Context, object runtime.Object, tableOptions runtime.Object) (*metav1.Table, error) {
 	return rest.NewDefaultTableConvertor(Resource("organizations")).ConvertToTable(ctx, object, tableOptions)
```

Comment:
This `organizations` should be `externalOrganizationResource`?

Reply:
done